src/test/java/neculaclaudiu/tests/testNg/CuraHealthcareTesttt.java

```diff
@@ -0,0 +1,105 @@
+package neculaclaudiu.tests.testNg;
+
+import com.beust.ah.A;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+@Test
+public class CuraHealthcareTesttt extends BaseTest{
```

Comment:
Please remove the extra 't's from name of the class at the end. Leave only one 't'. CuraHealthcareTesttt -> CuraHealthcareTest. Thanks!